packages/core/android/src/main/java/com/datadog/reactnative/DdTrace.kt

```diff
@@ -32,8 +39,17 @@ class DdTrace(reactContext: ReactApplicationContext) : ReactContextBaseJavaModul
      */
     @ReactMethod
     fun startSpan(operation: String, context: ReadableMap, timestampMs: Double, promise: Promise) {
-        val result = nativeInstance.startSpan(operation, context.toHashMap(), timestampMs.toLong())
-        promise.resolve(result)
+        val span = tracer.buildSpan(operation)
```

Comment:
can it happen to have concurrent calls in RN ? In that case maybe you want to synchronize that `spanMap`

Reply:
We can have successive calls made to native functions, but the JS is monothreaded, and the bridge between native and js can only process one message at a time.

However I am not 100% sure it is impossible to have a race condition in the native side.

I would say the same thing that for other changes in Android: let's wait for the tests before making this change